packages/core/test/ObjectField_test.js

```diff
@@ -414,7 +414,7 @@ describe("ObjectField", () => {
         formData: { first: 1 },
       });
 
-      expect(node.querySelectorAll(".field-string")).to.have.length.of(1);
+      expect(node.querySelectorAll(".field-string")).to.have.length.of(2);
```

Comment:
Why does this change from 1 to 2?

Reply:
This change adds a field-string wrapper for a single input. I'll investigate how to remove the wrapper shortly

Reply:
Turned out the wrapper is added here. We also cannot use `{ type: "string", [ADDITIONAL_PROPERTY_FLAG]: {} }}` as the flag requires many additional attributes which are not in arguments.
https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/fields/SchemaField.js#L197-L199

Reply:
Sorry, I'm still a bit confused as to why we are seeing this behavior only with this PR. This schema doesn't have anything to do with additionalProperties, right?

Can you explain a bit further -- or if you want to quickly grab some time to talk over Zoom, you can also do that (https://calendly.com/aramaswamis/meeting?month=2021-09)?